web/ui/react-app/src/pages/targets/TargetLabels.test.tsx

```diff
@@ -21,32 +20,32 @@ describe('SeriesLabels', () => {
     idx: 1,
     scrapePool: 'cortex/node-exporter_group/0',
   };
-  const seriesLabels = shallow(<SeriesLabels {...defaultProps} />);
+  const targetLabels = shallow(<TargetLabels {...defaultProps} />);
 
   it('renders a div of series labels', () => {
-    const div = seriesLabels.find('div').filterWhere(elem => elem.hasClass('series-labels-container'));
+    const div = targetLabels.find('div').filterWhere(elem => elem.hasClass('series-labels-container'));
     expect(div).toHaveLength(1);
     expect(div.prop('id')).toEqual('series-labels-cortex/node-exporter_group/0-1');
   });
 
-  it('wraps each label in a SeriesLabel', () => {
+  it('wraps each label in a label badge', () => {
     const l: { [key: string]: string } = defaultProps.labels;
-    Object.keys(l).forEach((key: string): void => {
-      const label = seriesLabels.find(SeriesLabel).filterWhere(label => label.prop('labelName') === key);
-      expect(label.prop('labelValue')).toEqual(l[key]);
+    Object.keys(l).forEach((labelName: string): void => {
+      const badge = targetLabels.find(Badge).filterWhere(badge => badge.hasClass(labelName));
+      expect(badge.children().text()).toEqual(`${labelName}="${l[labelName]}"`);
     });
-    expect(seriesLabels.find(SeriesLabel)).toHaveLength(3);
+    expect(targetLabels.find(Badge)).toHaveLength(3);
   });
 
   it('renders a tooltip for discovered labels', () => {
-    const tooltip = seriesLabels.find(Tooltip);
+    const tooltip = targetLabels.find(Tooltip);
     expect(tooltip).toHaveLength(1);
     expect(tooltip.prop('isOpen')).toBe(false);
     expect(tooltip.prop('target')).toEqual('series-labels-cortex/node-exporter_group/0-1');
     expect(tooltip.prop('isOpen')).toBe(false);
```

Comment:
Is it needed isOpen to be checked twice? Btw you can use ```toBeFalsy()```

Reply:
Good catch, thanks. I prefer using `toBe(false)` because it is a stronger and more precise assertion than `toBeFalsy()` which would also accept 0, null, etc.